2024-07-29 10:48:08,426 INFO     
Heat Source Version:  9.0.0b28


Oregon Department of Environmental Quality

This program comes with ABSOLUTELY NO WARRANTY. Appropriate model 
use and application are the sole responsibility of the user. 
This is free software, and you are welcome to redistribute it under 
certain conditions described in the License.


2024-07-29 10:48:08,426 INFO     Starting Model Initialization
2024-07-29 10:48:08,426 INFO     Reading control file
2024-07-29 10:48:08,426 ERROR    Error: Traceback (most recent call last):
  File "C:\Users\RIPHAWK\anaconda3\envs\heatsource9\lib\site-packages\heatsource9\BigRedButton.py", line 278, in run
    HSP = ModelControl(model_dir, control_file, 0)
  File "C:\Users\RIPHAWK\anaconda3\envs\heatsource9\lib\site-packages\heatsource9\BigRedButton.py", line 99, in __init__
    self.HS = ModelSetup(model_dir, control_file, run_type)
  File "C:\Users\RIPHAWK\anaconda3\envs\heatsource9\lib\site-packages\heatsource9\ModelSetup\ModelSetup.py", line 57, in __init__
    self.inputs.import_control_file()
  File "C:\Users\RIPHAWK\anaconda3\envs\heatsource9\lib\site-packages\heatsource9\ModelSetup\Inputs.py", line 395, in import_control_file
    IniParams["datastart"] = timegm(strptime(IniParams["datastart"] + " 00:00:00", "%Y-%m-%d %H:%M:%S"))
  File "C:\Users\RIPHAWK\anaconda3\envs\heatsource9\lib\_strptime.py", line 562, in _strptime_time
    tt = _strptime(data_string, format)[0]
  File "C:\Users\RIPHAWK\anaconda3\envs\heatsource9\lib\_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '7/1/2023 00:00:00' does not match format '%Y-%m-%d %H:%M:%S'

